pkg/apis/spinnaker/v1alpha2/spinnakerservice_types.go

```diff
@@ -147,6 +149,30 @@ type SpinnakerServiceList struct {
 	Items           []SpinnakerService `json:"items"`
 }
 
+// ValidationSetting is the definition of the validation for a given setting
+type ValidationSetting struct {
+	Enabled          bool  `json:"enabled"`
+	FrequencySeconds int64 `json:"frequencySeconds"`
+	FailOnError      bool  `json:"failOnError"`
+}
+
+// SpinnakerValidation defines validation settings for the deployment
+type SpinnakerValidation struct {
+	ValidationSetting *ValidationSetting      `json:"validationSetting"`
```

Comment:
I expect `enabled`, `frequencySeconds` and `failOnError` to be at this level not under `validationSetting`.